alf/config_util_test.py

```diff
@@ -182,6 +182,63 @@ def sole_init_test_env(x):
         self.assertEqual(alf.get_config_value("sole_init_test_twice.x"), 1)
         self.assertEqual(alf.get_config_value("sole_init_test_env.x"), 1)
 
+        # Test override_config doesn't doesn't overwrite for immutable values.
+        @alf.configurable
+        def override_on_immutable(x):
+            pass
+
+        alf.config("override_on_immutable", x=0, mutable=False)
+        alf.override_config("override_on_immutable", x=1)
```

Comment:
Is it possible to raise an error on overriding immutable config?  Silently ignoring override can be hard to debug?

Reply:
Refer to my reply above for why this cannot be done currently without adding more state parameters. There is a logger warning that gets printed in this scenario, though it's easy for users to now see it.

Reply:
Maybe let's color code the warning message a bright yellow, so it's harder to miss?

Reply:
Right now it prints as red via `absl.logging.warning`. Do you think yellow is better?